scipy/special/_add_newdocs.py

```diff
@@ -7419,8 +7438,8 @@ def add_newdoc(name, doc):
 
     References
     ----------
-    .. [1] Grant, Boyd, and Ye, "CVX: Matlab Software for Disciplined Convex
-        Programming", http://cvxr.com/cvx/
+    .. [1] Grant, Boyd, and Ye, "The CVX Users' Guide",
+           http://web.cvxr.com/cvx/doc/CVX.pdf
```

Comment:
I'm concerned that user might think we mean the GPL-licensed code. @person142 LMK if the Users' Guide is not the reference you had in mind.

Reply:
Really not a fan of links like these that break all the time and we don't know about it. But if there is really no DOI we can refer to.

Reply:
One possibility is that we could cite Boyd’s Convex Optimization book, which does have a stable doi and also gives definitions for these functions. 

https://doi.org/10.1017/CBO9780511804441

Reply:
Done.